src/extension/lsp/closing_labels_decorations.ts

```diff
@@ -41,6 +51,16 @@ export class LspClosingLabelsDecorations implements vs.Disposable {
 			const filePath = fsPath(td.uri);
 			delete this.closingLabels[filePath];
 		}));
+		this.subscriptions.push(vs.workspace.onDidChangeConfiguration((e) => {
+			if (e.affectsConfiguration("dart.closingLabels") || e.affectsConfiguration("dart.closingLabelsPrefix") || e.affectsConfiguration("dart.closingLabelsTextStyle")) {
```

Comment:
I don't think we need to check the prefix here, since it's not part of the decoration type.

Reply:
It is part of the configurations. So we don't need to "get" that value every time. If you disagree I can move it back or at least separate the `if`s.

Reply:
It could be something like:

```ts
let updatedWatched = false;
if (e.affectsConfiguration("dart.closingLabelsPrefix")) {
    updatedWatched = true;
    this.closingLabelsPrefix = config.closingLabelsPrefix;
}
if (e.affectsConfiguration("dart.closingLabels") || e.affectsConfiguration("dart.closingLabelsTextStyle")) {
    updatedWatched = true;
    this.decorationType.dispose();
    this.buildDecorationType();
}
if (updatedWatched) {
    this.update();
}
```